.gitignore

```diff
@@ -36,6 +36,3 @@ yarn-error.log*
 
 .eslintcache
 *.tsbuildinfo
-
-*.tsbuildinfo
-*.tsbuildinfo
```

Comment:
maybe delete this extra change here. Is not necesary

Reply:
Reverted it